### PR TITLE
[DESK-481][DESK-290][DESK-479][DESK-474][DESK-473][DESK-454][DESK-466][DESK-488][DESK-483][DESK-426][DESK-471][DESK-480][DESK-480] Grouped tickets

### PR DESCRIPTION
**Pull request by JamieRuderman**
### Description

    DESK-481 raspberry pi name
    DESK-290 disable auto-update
    DESK-479 default service types
    DESK-474 dismiss success snackbar
    DESK-473 missing services label
    DESK-454 better uninstall instructions
    DESK-466 breadcrumbs in reverse order
    DESK-488 launch ssh dialog button broken
    DESK-483 improved default service type selections
    DESK-426 show message when the max services have been added
    DESK-471 show auto-update toggle
    DESK-480 Added support for clear all credentials
    DESK-480 Checking for logged out cli root user
    Headless service script execute permissions
    Moved -j flag to first param
    Update CLI version
    Add ability to download from AWS
    Populate up cli errors to client
    2.5.24-alpha.6 tag

### Checklist

- [x] Pull Request title is in the format `[PROJ-12345] Some useful description here...`
- [ ] ~~I have prefixed this PR with `[WIP]` if it is a Work In Progress (not ready to merge)~~
- [ ] ~~The git branch is in the format `PROJ-12345-some-useful-description-here`~~
- [x] I have added one or more reviewers
- [x] I have added/updated tests for any code changes
- [ ] ~~I have updated documentation (including README, inline comments and docs.remote.it docs)~~
- [ ] ~~This PR only contains one isolated change (bug fix, feature, etc)~~
- [ ] ~~Translation strings added/updated for all user-visible text~~
- [x] I have manually reviewed this PR to make sure only the files I intended to change were changed and nothing else
- [x] I have notified reviewers on Slack by @ mentioning them in the `#desktop`channel
- [x] I have moved the Jira ticket into `Resolved` state and made a note in the ticket with a link to this PR

### Ticket

<https://remoteit.atlassian.net/browse/DESK-481>
<https://remoteit.atlassian.net/browse/DESK-290>
<https://remoteit.atlassian.net/browse/DESK-479>
<https://remoteit.atlassian.net/browse/DESK-474>
<https://remoteit.atlassian.net/browse/DESK-473>
<https://remoteit.atlassian.net/browse/DESK-454>
<https://remoteit.atlassian.net/browse/DESK-466>
<https://remoteit.atlassian.net/browse/DESK-488>
<https://remoteit.atlassian.net/browse/DESK-483>
<https://remoteit.atlassian.net/browse/DESK-426>
<https://remoteit.atlassian.net/browse/DESK-471>
<https://remoteit.atlassian.net/browse/DESK-480>

### CODE REVIEW

File: backend/src/Command.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ export default class Command {
           error: stderr.toString(),
         })
         // Hiding these errors for now because successful CLI commands are in stderr
```
Comment (bstrech) [inline backend/src/Command.ts:60]: Shouldn't this comment go away?

File: frontend/src/components/LocalhostScanForm/LocalhostScanForm.tsx [hunk 2]
```diff
@@ -28,42 +29,53 @@ export const LocalhostScanForm: React.FC<Props> = ({ setSelected }) => {
...
   }, [])
 
-  const updateTargets = useCallback(() => {
-    const selected = scanData.filter((_, key) => state[key])
-    setSelected(selected)
-  }, [scanData, setSelected, state])
+  const updateTargets = useCallback(
+    checked => {
+      const selected = scanData.filter((_, key) => checked[key])
+      console.log('selected', selected)
```
Comment (bstrech) [inline frontend/src/components/LocalhostScanForm/LocalhostScanForm.tsx:35]: Do you mean to keep this console.log here?

File: frontend/src/components/LocalhostScanForm/LocalhostScanForm.tsx [hunk 3]
```diff
@@ -28,42 +29,53 @@ export const LocalhostScanForm: React.FC<Props> = ({ setSelected }) => {
...
-    if (scanData && scanData.length !== state.length) {
-      state.length = scanData.length
-      state.fill(true)
-      updateTargets()
-    }
-  }, [scanData, state, updateTargets])
+  // useEffect(() => {
+  if (scanData && scanData.length !== state.length) {
+    const checked = scanData.map(row => !!serviceTypes.find(st => st.defaultPort === row.port || 29999 === row.port)) // 29999 temp hack to have remoteit admin checked
+    console.log('CHECKED', checked)
```
Comment (bstrech) [inline frontend/src/components/LocalhostScanForm/LocalhostScanForm.tsx:44]: console.log still needed?

File: backend/src/Command.ts (hunk 1 above)
Comment (JamieRuderman) [reply backend/src/Command.ts:60]: Yes.

File: frontend/src/components/LocalhostScanForm/LocalhostScanForm.tsx (hunk 2 above)
Comment (JamieRuderman) [reply frontend/src/components/LocalhostScanForm/LocalhostScanForm.tsx:35]: Nope good catch.

File: frontend/src/components/LocalhostScanForm/LocalhostScanForm.tsx (hunk 3 above)
Comment (JamieRuderman) [reply frontend/src/components/LocalhostScanForm/LocalhostScanForm.tsx:44]: Thanks!

**Pull request merged**